src/specs/components/checkbox/demo/data--eu.js

```diff
@@ -9,7 +9,8 @@ module.exports = {
   name: 'checkbox-default',
   invalid_icon: {
     path: '/icons.svg',
-    name: 'warning',
+    type: 'notifications',
```

Comment:
If I'm not wrong, we don't need to have this 'type' field anymore, all icons are in the same category now (same for the other data files)

Reply:
cool thanks for the info, as i saw some specified i wanted to align them all up ;) i removed them everywhere..